packages/ui-components/src/__tests__/DeploymentSteps.test.ts

```
@@ -1,30 +1,62 @@
+	orderbookAddress: '0x456' as Hex,
+	chainId: 1337,
+	network: 'testnet'
+};
+
+const mockDeploymentHandlers: DeploymentHandlers = {
+	handleDisclaimerModal: vi.fn(),
+	handleDeployModal: vi.fn()
+};
+
+const defaultProps = {
+	dotrain,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)